### ISSUE TITLE
Proposal: filemanager icon

### ISSUE THREAD

**Issue by Feichtmeier** [ubuntu/yaru#1755]
I think I did not make a secret out of my opinion on the blue file cabinet icon
![image](https://user-images.githubusercontent.com/15329494/72531631-cd946f00-3869-11ea-8929-bf47fee28dcc.png)

I really think it does not fit into that whole origami style of all the other nice icons you've made @ubuntujaggers 

I spoilered the idea  on telegram but here - for the "rest" - this is the idea:

![image](https://user-images.githubusercontent.com/15329494/72531810-2bc15200-386a-11ea-9afb-328a2f3fd545.png)

Basically the "old" / Suru icon but on one of the new Yaru baseplates.
I am not sure about the brown though, but it's prbly a safe pick.

Or a white one:
![image](https://user-images.githubusercontent.com/15329494/72532151-c883ef80-386a-11ea-884b-4cc6cade02ca.png)

Or... *insert any other baseplate color here*

Opinions, onions and whatnot? @madsrh @ubuntujaggers @clobrano ?

Just for the history on why we changed it: we completely removed the baseplate back then and it was only a single white folder as an icon - then someone complained (and this was a correct complain) that it would only represent a directory and not a file manager)

**Comment by clobrano** [comment]
All the "complains" still hold for me:
- the folder icon was common under Ubuntu, but someone pointed out that it looks like a real folder, which is not
- the cabinet icon was not common under Ubuntu, so someone -at first- did not notice nautilus was there, just with a different icon

I am okay with both really, my only note is to take one and be consistent, better not going back and forth every other release :)

**Comment by Feichtmeier** [comment]
Or make the current one look more like the rest of the icons :man_shrugging: 
Most of them only have 1 or 2 basic symbol on them. Very simple. I think the current one is too much. Too many highlights too many elements (3 handles 3 drawers) which make it look more "glossy" than the other icons.
I am not saying this is the best fit - I just do not think the current one is one of the better ones.

@clobrano most suru icons give the "context" by putting a symbol on a baseplate. 
Baseplate: square
Symbol: folder

**Comment by clobrano** [comment]
Really, my only point was to decide for one - old or new - if we all are happy and can live long with it, or keep the current otherwise, I am more than open to proposals.

**Comment by Feichtmeier** [comment]
Ok great! Just to get it more in line with the suru/yaru icons and to not some ideas on how those icons are made:

- they clearly should remind of paper or concrete: origami
- They mostly only feature either one symbol or the whole icon merges with the symbol
- every icon is an abstraction
- common symbols, symbols than people know the meaning of (a symbol needs a concrete connection to reality or it is not a symbol by definition), this means if people understand the meaning - like for example a folder on a piece of paper - you've won

**Comment by ubuntujaggers** [comment]
Reshaping the original Suru icon to use one of our new baseplates would be easy - should I just make a PR so we can try it?

**Comment by ubuntujaggers** [comment]
I liked the old folder design, I think we mostly changed to a filing cabinet due to suggestions from others? Happy to give this a whirl 👍

**Comment by chrisjbillington** [comment]
I like the current one because it's distinctive - it's a distinctive colour and doesn't look like anything else.

I was under the impression there had been a shift away from uniformity of icons in GNOME (not sure about Yaru actually) toward more heterogeneity, party due to the desire for more distinctiveness between icons, and partly out of practicality so that icons that are not part of a set don't stick out like a sore thumb.

Anyway, +1 for blue filing cabinet FWIW!

**Comment by Feichtmeier** [comment]
Fixed by https://github.com/ubuntu/yaru/pull/1756

**Comment by Feichtmeier** [comment]
Reopened for further discussion

**Comment by ubuntujaggers** [comment]
In the post squircle world, we've had three iterations of this icon, and I like them all:

- Full sized folder
- Blue filing cabinet
- Small folder on backplate

The filing cabinet has the advantage of closely resembling Adwaita:

![image](https://user-images.githubusercontent.com/38893390/72834410-ae8a4880-3c80-11ea-823b-3b0697928d95.png)

...but whatever we go for has to be "safe" (in terms of no one really dislikes it) because it's a key app and goes on the launcher by default.

**Comment by coxackie** [comment]
I would vote for fill sized-folder, but I am happy with all. (By the way, are you guys interested in designing icons outside of the core gnome apps? If yes, I (or you) could open an app suggestions discussion if you wish.)

**Comment by clobrano** [comment]
We are interested as long as the application maintainers agree and are
involved as well

**Comment by Feichtmeier** [comment]
Files / Nautilus is a generic app and a gnome core app. I don't think the we need an ack here

We should focus on the best design imho

**Comment by ubuntujaggers** [comment]
How about this - warmer and less dark grey, with a bit of work on the inside of the folder to make it more pronounced (without resorting to a high contrast) in the full size version:

![image](https://user-images.githubusercontent.com/38893390/72894846-18076700-3d14-11ea-9f02-ce2a6eab7d0e.png)

![image](https://user-images.githubusercontent.com/38893390/72894869-205fa200-3d14-11ea-956c-815ece67f8c5.png)

![image](https://user-images.githubusercontent.com/38893390/72894887-2b1a3700-3d14-11ea-9897-6a4e0b639d79.png)

**Comment by friendlyapple** [comment]
Sorry but I prefer the blue file cabinet icon, more easily recognizable and adding more variety in the Yaru icons.

**Comment by Feichtmeier** [comment]
We dont need more variety, we need to stick with the guidelines, or make a new guideline :)

**Comment by friendlyapple** [comment]
When I talk about variety, it's actually in terms of color. I think the idea of the blue file cabinet icon should not be discarded. I think there must be a way to make it a little more "Yaru".

**Comment by fitojb** [comment]
A huge step backwards.

**Comment by Amr-Ibra** [comment]
I also agree that this is a step backwards. The cabinet icon is a better representation of a file manager app. A brown folder on a baseplate looks just like a folder. Upstream Nautilus icon is also a cabinet.

**Comment by Feichtmeier** [comment]
![Screenshot_20200129-224614](https://user-images.githubusercontent.com/15329494/73400384-4b845b80-42e9-11ea-9c18-8f5cf1ee8ba8.png)

**Comment by Feichtmeier** [comment]
That's Android 10
iOS looks the same
Everyone and his mom knows that this is the file manager

**Comment by ubuntujaggers** [comment]
I think the folder metaphor is widespread enough that it's unambiguous in practice. It seems to be the most common choice (I know Windows includes a little holder but I wonder how many people know that's what it is). I think first time users wouldn't quickly guess that the filing cabinet is a file manager, a) because it's not seen in other OSes and b) because it's blue, which filing cabinets rarely are?

**Comment by Feichtmeier** [comment]
Exactly. No operating system uses the file cabinet. So the idea of a that metaphor being more obvious is false. Let's move on

**Comment by fitojb** [comment]
Please. Be. Coherent. You supposedly agreed in the past to:
1. Follow upstream GNOME conventions to avoid BS PR disasters like the whole “Don’t theme my app”
2. Avoid the use of background plates to make icons’ shapes different and distinctive and ease Yaru’s visual integration with third-party applications.

You now changed your mind 180 degrees on both these things, one release later?

**Comment by Feichtmeier** [comment]
Regarding 1):

We avoid those disasters by

- only changing pre installed app's icons, or app icons where we explicitly asked the devs for there okay
- only changing generic gnome application app icons from the list of gnome core apps, where the gnome designers and/or gnome leaders are okay with changing the icons

We still use the Suru icon guidelines. And the old icon did not fit into the guidelines

I didn't change my mind. You need to keep those two topics separated from each other

**Comment by fitojb** [comment]
> And the old icon did not fit into the guidelines

LOL, now you’re accusing @ubuntujaggers of not following his own guidelines, haha.

**Comment by chrisjbillington** [comment]
A brown/cream folder in a squircle is just so ... indistinctive. Here it is on my taskbar (dash to panel). It's a featureless, colourless blob, and easily the least distincitve or recognisable icon on there.

![image](https://user-images.githubusercontent.com/1044087/73466662-3393f280-4350-11ea-9a3b-cbe76afd3bef.png)

It's true that a folder is the most recognisable icon for a file browser, but what about a blue (or otherwise colourful) folder, and losing the squircle?

If the new folder icons weren't grey, I would propose making the 'files' icon identical to the 'home folder' icon. It would make more sense for an icon based on a folder to actually be the same as the yaru folder icon:

![image](https://user-images.githubusercontent.com/1044087/73466594-12330680-4350-11ea-92da-80fd8c1d9cc9.png)

It being the 'home folder ' icon specifically would make sense since the file browser by default opens your home folder. But grey is even worse than brown and cream on taskbars - the purple edge of the folder icon saves it a bit, but not much. As I've complained about elsewhere (though others seem to not agree with me), the grey folder icons are low-contrast on dark backgrounds generally:

![image](https://user-images.githubusercontent.com/1044087/73466979-a69d6900-4350-11ea-9137-20b390065eee.png)

If you want to make the nautilus icon a folder, I wouldn't mind that, but I think the following would be better than the current arrangement:

* Folder icons are a distinct colour like blue or orange (or even purple!)
* Nautilus's icon is identical to the home folder icon, no squircle or anything.

**Comment by clobrano** [comment]
Let's calm down, everybody here has the same interest in making Yaru better. No need to laugh or get angry at each other replies.

Coherence is not in discussion. As said, we agreed on not using the same background for all the icons, and we agreed in changing the icons for a subset of applications. This is not an opinion.

Constructive feedbacks are most welcome

@chrisjbillington, it's good to have different perspectives. We considered only the default Ubuntu dock as reference, where the size is 48px and the icon is pretty easy to recognize.

![image](https://user-images.githubusercontent.com/2883614/73468356-f1d16f00-4384-11ea-808d-a694d6e5348c.png)

**Comment by friendlyapple** [comment]
I agree with @clobrano and @Feichtmeier. It is difficult for everyone to agree. We don't have the same tastes (the proof, I think like you, I prefer the blue file cabinet). But this conversation may have gone a little too far and some sentences are unfair. @madsrh, @clobrano, @ubuntujaggers and @Feichtmeier are doing a colossal work to maintain this project (and take into account all comments) and actively contribute. It is thanks to them that we have Yaru, the theme that we all love. I hope they can continue the work they do for a long time.

**Comment by fcole90** [comment]
I think the cabinet looks more distinctive and also it would be in line with both the current Gnome design and the former Ubuntu style

![image](https://user-images.githubusercontent.com/1292230/74132772-de799b80-4bef-11ea-9b2e-ef415f08ac5f.png)

![image](https://user-images.githubusercontent.com/1292230/74132801-edf8e480-4bef-11ea-8321-d0339016ca66.png)

I can also understand your concerns that it looks too much detailed and less origami than the others, but I would rather suggest iterating on the cabinet than having just the old folder.

Also, if you really prefer a folder design, I think that indeed it should not _just_ be the former icon. It feels very out of place with the current restyle of the other icons. Perhaps just the folder icon (as was proposed) but with some distinctive colour or symbol could work.

In summary, while I'm more in favour of the cabinet, a folder design would be fine but just not the Suru one. 

Extra - Icons from other desktop OS:
![image](https://user-images.githubusercontent.com/1292230/74133192-b179b880-4bf0-11ea-8195-04361d3dde32.png)
![image](https://user-images.githubusercontent.com/1292230/74133281-ec7bec00-4bf0-11ea-9bf8-9c062a305b84.png)
![image](https://user-images.githubusercontent.com/1292230/74133339-0b7a7e00-4bf1-11ea-9b84-c7ab87bf9d0e.png)
